test/bdd/pkg/v1/vc/credential.go

```diff
@@ -222,10 +248,30 @@ func (e *Steps) getVerificationResult(
 	createdCredential := e.bddContext.CreatedCredential
 	e.RUnlock()
 
-	cred, err := verifiable.ParseCredential(createdCredential, verifiable.WithDisabledProofCheck(),
-		verifiable.WithJSONLDDocumentLoader(loader))
-	if err != nil {
-		return nil, err
+	var cred *verifiable.Credential
+	if vcsverifiable.IsSDJWT(string(createdCredential)) {
+		// SD-JWT case
+		cfi := common.ParseCombinedFormatForIssuance(string(vcsverifiable.UnQuote(createdCredential)))
+
+		cred, err = verifiable.ParseCredential([]byte(cfi.SDJWT), verifiable.WithDisabledProofCheck(),
+			verifiable.WithJSONLDDocumentLoader(loader))
+		if err != nil {
+			return nil, err
+		}
+
+		cfp, err := holder.CreatePresentation(string(vcsverifiable.UnQuote(createdCredential)), cfi.Disclosures)
```

Comment:
For SD JWT credential verification I'm using combined format for presentation.

Reply:
For BDD test you can create presentation by adding ~ at the end of combined format for issuance

Reply:
technically yes, but the idea was to follow the [API specification](https://github.com/trustbloc/vcs/blob/main/docs/v1/openapi.yaml#L780).
Ok, updated